.github/workflows/build.yml

```diff
@@ -16,27 +16,13 @@ concurrency:
 jobs:
   build-binaries:
     name: Build binaries
-    runs-on: runs-on,runner=4cpu-linux-${{ matrix.arch }},hdd=50,run-id=${{ github.run_id }}
-    strategy:
-      matrix:
-        arch: [x64, arm64]
+    runs-on: runs-on,runner=4cpu-linux-x64,hdd=50,run-id=${{ github.run_id }}
     steps:
     - name: Checkout code
       uses: actions/checkout@v4
 
-    # Build binaries
     - name: Run make ci
-      if: ${{ matrix.arch == 'x64' }}
       run: make ci
-    - name: Run make arm
-      if: ${{ matrix.arch == 'arm64' }}
```

Comment:
Does the change mean the arm path is not tested in PRs?

Reply:
Added it back. Yeah, we need to run it in PR.